compiler/aarch64/codegen/ARM64Debug.cpp

```diff
@@ -1128,6 +1226,31 @@ TR_Debug::print(TR::FILE *pOutFile, TR::ARM64Trg1Src2Instruction *instr)
    trfflush(_comp->getOutFile());
    }
 
+void
+TR_Debug::print(TR::FILE *pOutFile, TR::ARM64ZeroSrc2Instruction *instr)
+   {
+   printPrefix(pOutFile, instr);
+   TR::InstOpCode::Mnemonic op = instr->getOpCodeValue();
+   if (op == TR::InstOpCode::subsx || op == TR::InstOpCode::subsw)
+      {
+      // cmp alias
+      trfprintf(pOutFile, "cmp%c \t", (op == TR::InstOpCode::subsx) ? 'x' : 'w');
+      }
+   else if (op == TR::InstOpCode::andsx || op == TR::InstOpCode::andsw)
```

Comment:
You need to add lines for `addsx`/`addsw` for `cmnx`/`cmnw`.

Reply:
It was missing in the original `TR_Debug::print()` for `ARM64Trg1Src2Instruction`.  It is not your fault:
https://github.com/eclipse/omr/blob/b68a7d3001d610b2002cea1e3fca00b94e44bd12/compiler/aarch64/codegen/ARM64Debug.cpp#L1106-L1116

Reply:
Added a block for `addsx`/`addsw`.